components/gitpod-cli/cmd/idp-gcloud-token.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/spf13/cobra"
+)
+
+var idpGCloudTokenOpts struct {
+	Audience []string
+}
+
+var idpGCloudTokenCmd = &cobra.Command{
+	Use:   "gcloud-token",
+	Short: "Requests an gcloud format ID token for this workspace",
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		cmd.SilenceUsage = true
+		if len(idpGCloudTokenOpts.Audience) == 0 {
+			return fmt.Errorf("missing --audience or GOOGLE_EXTERNAL_ACCOUNT_AUDIENCE env var")
```

Comment:
Why we need env `GOOGLE_EXTERNAL_ACCOUNT_AUDIENCE` and `GOOGLE_EXTERNAL_ACCOUNT_OUTPUT_FILE`? Could you input more context?